src/Trigger/RabbitMQListener.cs

```diff
@@ -59,6 +60,7 @@ public Task StartAsync(CancellationToken cancellationToken)
 
             _consumer.Received += (model, ea) =>
             {
+                _channel.BasicAck(ea.DeliveryTag, true);

```

Comment:
is this an intentional change for this PR? Will you be addressing Acks on a separate PR?

Reply:
I just included it as a part of this PR because the samples don't work as expected otherwise. But more in depth changes would be made in a separate PR